src/state/opamSysInteract.ml

```diff
@@ -372,22 +376,22 @@ let install_packages_commands sys_packages =
     let epel_release = "epel-release" in
     let install_epel rest =
       if List.mem epel_release packages then
-        ["yum", ["install"; epel_release]] @ rest
+        ["yum", "install"::yes ["-y"] [epel_release]] @ rest
       else rest
     in
     install_epel
-      ["yum", "install"::
-              (OpamStd.String.Set.of_list packages
-               |> OpamStd.String.Set.remove epel_release
-               |> OpamStd.String.Set.elements);
+      ["yum", "install"::yes ["-y"]
+                (OpamStd.String.Set.of_list packages
+                 |> OpamStd.String.Set.remove epel_release
+                 |> OpamStd.String.Set.elements);
        "rpm", "-q"::"--whatprovides"::packages]
-  | Debian -> ["apt-get", "install"::packages]
-  | Freebsd -> ["pkg", "install"::packages]
-  | Gentoo -> ["emerge", packages]
-  | Homebrew -> ["brew", "install"::packages]
-  | Macports -> ["port", "install"::packages]
-  | Openbsd -> ["pkg_add", packages]
-  | Suse -> ["zypper", ("install"::packages)]
+  | Debian -> ["apt-get", "install"::yes ["-qq"; "-yy"] packages]
```

Comment:
Another option here, which is less aggressive, is `DEBIAN_FRONTEND=noninteractive`
(note that this could also be manually set by the caller as a workaround).

Reply:
In #4200, I add the handling of giving environment variables to install commands.